tests/binary-indirect-cbits/BUILD

```diff
@@ -7,17 +7,16 @@ load(
 haskell_binary(
     name = "binary-indirect-cbits",
     srcs = ["Main.hs"],
-    linkstatic = False,
     deps = [
         "//tests/library-with-cbits",
         "@hackage//:base",
     ],
+    linkstatic = False
 )
 
 haskell_binary(
     name = "binary-indirect-cbits-partially-static",
     srcs = ["Main.hs"],
-    linkstatic = False,
```

Comment:
Can we actually set `linkstatic` to `True` here, doesn’t that change the meaning of the test?

Reply:
AFAIK `True` is the default value.

Reply:
Indeed, this shouldn't be removed. I'll add it back and rebase the PR.